GCC 14 fixes (not sure why this is no needed for Firefox itself).

--- thunderbird-128.1.0/media/ffvpx/libavcodec/av1dec.c
+++ thunderbird-128.1.0/media/ffvpx/libavcodec/av1dec.c
@@ -887,7 +887,10 @@
         ff_cbs_fragment_reset(&s->current_obu);
     }
 
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wincompatible-pointer-types"
     s->dovi.logctx = avctx;
+#pragma GCC diagnostic pop
     s->dovi.dv_profile = 10; // default for AV1
     sd = ff_get_coded_side_data(avctx, AV_PKT_DATA_DOVI_CONF);
     if (sd && sd->size > 0)
--- thunderbird-128.1.0/media/ffvpx/libavcodec/libdav1d.c
+++ thunderbird-128.1.0/media/ffvpx/libavcodec/libdav1d.c
@@ -289,7 +289,10 @@
     c->delay = res > 1 ? res : 0;
 #endif
 
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wincompatible-pointer-types"
     dav1d->dovi.logctx = c;
+#pragma GCC diagnostic pop
     dav1d->dovi.dv_profile = 10; // default for AV1
     sd = ff_get_coded_side_data(c, AV_PKT_DATA_DOVI_CONF);
     if (sd && sd->size > 0)
